protected/controllers/UserController.php

```diff
@@ -57,7 +57,7 @@ public function actionAutoComplete($term, $consultant_only = false)
 				$res[] = array(
 					'id' => $user->id,
 					'label' => $user->getFullNameAndTitle(),
-					'value' => $user->getFullName(),
```

Comment:
If this is an existing function, then it should not be changed. We don't know what else this change could affect.